kubernetes/samples/charts/voyager/README.md

```diff
@@ -0,0 +1,91 @@
+# Install and Configure Voyager
+
+## Step-by-step Guide to Install Voyager Operator
+Appscode has provided helm chart to install Voyager. See official installation document: https://appscode.com/products/voyager/7.4.0/setup/install/.
+
+As a demonstration, following are the detail steps to install Voyager operator using helm chart in hostlinux.
```

Comment:
As a demonstration, the following are the detailed steps to install the Voyager operator by using a Helm chart in a Linux host.